recipes/flake8/meta.yaml

```diff
@@ -1,63 +0,0 @@
-{%set name = "flake8" %}
```

Comment:
I think I can also create a different branch, and make a pull request from there. Will that be fine?